src/geometry/manifold/derivation_bundle.lean

```diff
@@ -0,0 +1,163 @@
+/-
+Copyright © 2020 Nicolò Cavalleri. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Nicolò Cavalleri
+-/
+
+import geometry.manifold.algebra.smooth_functions
+import ring_theory.derivation
+
+/-!
+
+# Derivation bundle
+
+In this file we define the derivations at a point of a manifold on the algebra of smooth fuctions.
+Moreover we define the total bundle of derivations (although at the moment it has not been given a
```

Comment:
```suggestion
Moreover we define the total bundle of derivations (although as of May 2021 it has not been given a
```

Reply:
```suggestion
Moreover we define the total bundle of derivations (although as of June 2021 it has not been given a
```

Reply:
The point is that if it will be given a topology it will happen in this same file, so it is very easy to check whether or not it has a topology and to update the docs...